stories/Cookiebar/Cookiebar.stories.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withA11y } from '@storybook/addon-a11y'
+import { withInfo } from '@storybook/addon-info'
+
+
+import Esempi from './docs/example.md'
+
+import { CookieBar,CookieBarButtons,CookieBarButton } from '../../src'
+
+class CookieBarComponent extends React.Component {
```

Comment:
Change it to a Function component here: the problem is that classes are not correctly shown in the source code panel, so this code will be unreachable for the user.
There's effectively no need to have a state here if we can manage it elsewhere.
The state will be handled by knob rather than the user click on the component.

Reply:
>handled by knob

do you mean [this](https://www.npmjs.com/package/@storybook/addon-knobs) ?

Reply:
See how they are used in the `Header` component: https://github.com/italia/design-react-kit/blob/master/stories/Header/CenterHeader/basic.js#L15

Reply:
@dej611  okay, so i should add a `boolean('show',true)`. 
The Cookiebar has to appear and disappear on clicking the show checkbox in the knob section right ?

Reply:
yes